### PR TITLE
Update view extent only when needed, with the use of ol.extent.equals()

### PR DESCRIPTION
**Pull request by edellucien**

### CODE REVIEW

File: src/components/BasicMap/BasicMap.js [hunk 1]
```diff
@@ -221,7 +221,13 @@ class BasicMap extends Component {
       this.map.getView().setResolution(resolution);
     }
 
-    if (viewOptions && prevProps.viewOptions.extent !== viewOptions.extent) {
+    if (
+      (viewOptions && viewOptions.extent && !prevProps.viewOptions.extent) ||
+      (viewOptions &&
+        viewOptions.extent &&
```
Comment (oterral) [inline src/components/BasicMap/BasicMap.js]: the 2 1st params are the same in each condition.

**Comment by oterral** [comment]
@edellucien conflict

**Comment by edellucien** [comment]
Conflict and comment resolved

File: src/components/BasicMap/BasicMap.js [hunk 2]
```diff
@@ -225,16 +226,22 @@ class BasicMap extends Component {
...
-          ...{ resolution },
-          ...{ extent: viewOptions.extent },
-        }),
-      );
+    if (viewOptions && viewOptions.extent) {
+      if (
+        !prevProps.viewOptions.extent ||
+        (prevProps.viewOptions.extent &&
+          !equals(prevProps.viewOptions.extent, viewOptions.extent))
+      ) {
```
Comment (oterral) [inline src/components/BasicMap/BasicMap.js]: In one if :

if (viewOptions && viewOptions.extent &&
    (!prevProps.viewOptions.extent ||
￼	        (prevProps.viewOptions.extent &&
￼	          !equals(prevProps.viewOptions.extent, viewOptions.extent))

**Comment by oterral** [comment]
@lucien new comment

**Comment by Lucien** [comment]
> @Lucien new comment

Oh no!

**Pull request merged**